docs/how_to_guides/validation/how_to_trigger_slack_notifications_as_a_validation_action.rst

```diff
@@ -1,16 +1,58 @@
 .. _how_to_guides__validation__how_to_trigger_slack_notifications_as_a_validation_action:
 
-How to trigger slack notifications as a Validation Action
+How to trigger Slack notifications as a Validation Action
 =========================================================
 
-.. admonition:: Admonition from Mr. Dickens
+This guide will help you trigger Slack notifications as a Validation Action.  If you have validation suites that take a very long time to run, it will allow you to receive a Slack message as to whether or not the Validation succeeded or failed.
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. admonition:: Prerequisites: This how-to guide assumes that you have already:
 
+    - Slack app with webhook function enabled (See Additional Resources for more information on how to set this up).
+    - Webhook address for your Slack app.
+    - Slack channel that messages will be sent to.
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
+Steps
+-----
+
+1. Open ``uncommitted/config_variables.yml`` file and add ``validation_notification_slack_webhook`` variable by adding the following line:
+
+.. code-block:: yaml
+
+    validation_notification_slack_webhook: [address to web hook]
+
+
+2. Open ``great_expectations.yml`` and add ``send_slack_notification_on_validation_result`` action to ``validation_operators``. Make sure the following section exists in the ``great_expectations.yml`` file.
+
+.. code-block:: yaml
+
+    validation_operators:
+        action_list_operator:
+            # To learn how to configure sending Slack notifications during evaluation
+            # (and other customizations), read: https://docs.greatexpectations.io/en/latest/reference/validation_operators/action_list_validation_operator.html
+            class_name: ActionListValidationOperator
+            action_list:
+            ...
+            # other actions
+            # here is what you will be adding
+            ...
+        - name: send_slack_notification_on_validation_result
```

Comment:
The formatting looks off here to me. Also, can you please add a short note to the effect of "The name can be set to any value."

Reply:
will do